Apps/Contoso.UWP.Demo/MainPage.xaml.cs

```diff
@@ -7,6 +7,7 @@
 using System.Threading.Tasks;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
+using UnhandledExceptionEventArgs = Windows.UI.Xaml.UnhandledExceptionEventArgs;
```

Comment:
What is this change?

Reply:
Resolving name conflict after target update. Was already approved in #1273